### DISCUSSION TITLE
Horizontal to Column container

### DISCUSSION THREAD

**Discussion by proton029** [Budibase/budibase#6855] [Help]
How can I make my horizontal container to column container so that my elements would be staked on top in mobile view. Tried media queries in embed but its not responding

**Comment by poirazis** [comment]
Hello @proton029 
You can Use the Conditional UI ( https://docs.budibase.com/docs/conditions ) to change the direction of your container.
Like so 
<img width="1467" alt="Screenshot 2022-07-22 at 12 54 15 PM" src="https://user-images.githubusercontent.com/80512170/180414964-1b7c9597-c38f-4869-9c76-180469aab219.png">
